packages/lodestar/test/e2e/sync/reqResp.test.ts

```diff
@@ -47,7 +46,7 @@ describe("[sync] rpc", function () {
     const chain = new MockBeaconChain({
       genesisTime: 0,
       chainId: 0,
-      networkId: new BN(0),
+      networkId: 0n,
```

Comment:
funny, why is network id bigint haha

Reply:
Not entirely sure... It was a BN, so just changed to bigint haha